modules/pam_faillock/main.c

```diff
@@ -201,8 +200,9 @@ do_allusers(struct options *opts)
 	for (i = 0; i < rv; i++) {
 		if (userlist[i]->d_name[0] == '.') {
 			if ((userlist[i]->d_name[1] == '.' && userlist[i]->d_name[2] == '\0') ||
-			    userlist[i]->d_name[1] == '\0')
-				continue;
+				userlist[i]->d_name[1] == '\0') {
+					continue;
+				}
```

Comment:
I don't mind adding curly brackets if you think it could help, but the new indentation is misleading.

Reply:
the original indentation is space, just convert that to tab, which can be told by the line 204.

Reply:
As you can see, `userlist[i]->d_name[1] == '\0'` is indented under `(userlist[i]->d_name[1] == '.' && userlist[i]->d_name[2] == '\0')`, so this conversion from spaces to tab breaks it.
Also, this conversion changes the indentation of `continue` from 4 tabs to 5 tabs, and indents the closing `}` one tab more that the matching `if` statement.  Both of these changes are misleading.
Could you leave this hunk unchanged, please?

Reply:
sure